test/dice_test.coffee

```diff
@@ -0,0 +1,27 @@
+Helper = require('../node_modules/hubot-test-helper/src/index.coffee')
+
+helper = new Helper('../scripts/dice.coffee')
+
+expect = require('chai').expect
```

Comment:
> The only case I can make

I mean the fact that it already can compare arrays without having to do anything is a good enough case in my :eyes:, so let's keep it.